script/test.py

```diff
@@ -28,13 +28,15 @@
 
 PROJECT_NAME = electron_gyp()['project_name%']
 PRODUCT_NAME = electron_gyp()['product_name%']
+PDF_VIEWER_ENABLED = electron_gyp()['enable_pdf_viewer%']
```

Comment:
Won't work if the value is changed via a command line argument =/

Reply:
One of the possible solutions would be to add an undocumented js API to get a feature availability.

```js
const {features} = require('electron')
features.isPdfViewerAvailable();  // false
```